conductr_cli/conduct_load.py

```diff
@@ -2,59 +2,64 @@
 from pyhocon.exceptions import ConfigMissingException
 from conductr_cli import bundle_utils, conduct_url, conduct_logging
 from functools import partial
+from urllib.parse import ParseResult, urlparse, urlunparse
+from urllib.request import urlretrieve
+from pathlib import Path
+
 import json
-import os
-import re
 import requests
 
-
 @conduct_logging.handle_connection_error
 @conduct_logging.handle_http_error
 @conduct_logging.handle_invalid_config
 @conduct_logging.handle_no_file
+@conduct_logging.handle_bad_zip
 def load(args):
     """`conduct load` command"""
 
-    if not os.path.isfile(args.bundle):
-        raise FileNotFoundError(args.bundle)
-
-    if args.configuration is not None and not os.path.isfile(args.configuration):
-        raise FileNotFoundError(args.configuration)
+    print('Downloading bundle...')
```

Comment:
`Downloading` is a bit confusing when loading a bundle from file system. Maybe change to `Retrieving bundle`?